Second set of nut fact and acrolinx updates (#4767)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/migrate-to-okta-prerequisites/main/index.md

This guide provides you with information to create a plan for migrating existing users to Okta.

## Prepare and plan for the migration

Migrating users from one system to another takes careful preparation and planning. A thorough understanding of your current environment and having clear migration goals provides the basis of a solid plan.

Also, being aware of any risks in your plan guides you toward successfully testing and deploying your migration.

## Migration types

A **one-time migration** moves your users with their credentials over to Okta all at once. It’s expected that the legacy system could (and should) be taken offline after this migration process completes.

> A **migration program** is one where, for a period of time, the legacy system and Okta run in parallel to migrate users in a more ad hoc fashion. You can accomplish this using the Okta [Inline Password Hook](/docs/reference/password-hook/) feature.

Reviewer: Can we use something other than "ad hoc" here?

* [How to Keep PII Secure while Migrating Users to the Cloud](https://www.okta.com/gdpr/) (Okta security blog post)
* [Prepare your organization for the GDPR](https://www.okta.com/gdpr/) (Okta data security page)

## Get ready to create a plan&mdash;gather information

> Before being able to create a plan, you must gather information about your environment and decide on specific goals of the migration project. This includes making a list of the types of users you want to migrate and the applications and other resources that they’ll need to access through Okta.

Reviewer: ```suggestion
Before being able to create a plan, you must gather information about your environment and decide on specific goals of the migration project. This includes making a list of the types of users you want to migrate and the apps and other resources that they’ll need to access through Okta.
```

### Identify the users to migrate

Start by identifying the types of users you have based on the broad kinds of access they need. For example:

* Full and part-time employees of your company
* Customers
* Partners
* Contractors

How many of each type of user do you have? Understanding this helps estimate the amount of time you need when implementing your migration.

Also identify any subgroups currently in use for all of your user types. For example, are your employees grouped into sales, marketing, and accounting groups? What about your customers?

### Locate your existing user profile data

> Your user data could be in Active Directory, LDAP, or any number of on-premise or cloud-based applications, but most importantly you need to know exactly where the data is and how to access it. This also requires that you have permission to access and use the data. You also need to know which system is the source of truth for each data element. For example, various systems might maintain a primary email address for each of your users.

Reviewer: ```suggestion
Your user data could be in Active Directory, LDAP, or any number of on-premise or cloud-based apps. Most importantly, you need to know exactly where the data is and how to access it. This also requires that you have permission to access and use the data. You also need to know which system is the source of truth for each data element. For example, various systems might maintain a primary email address for each of your users.
```

### Test your migration plan

Once you have a clean set of data, you can create a realistic set of test data that can be used to test your migration method. Testing uncovers issues with your data or the migration method so that issues can be addressed before going to production.

#### Create a test plan

> When creating a test plan, it’s important to prepare:

Reviewer: ```suggestion
When creating a test plan, it’s important to prepare the following:
```
